prepare_cyrus.hs

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env stack
+-- stack runghc --resolver lts-8.14 --package turtle --install-ghc
```

Comment:
Ok so I did that in haskell one because it was fast (I such in bash, and you don't have to run your script to see the error because of haskell strong typing), and I did not want to share it at first glace.
I automatized that because when I do manual task I always make mistake.

So now we have three options:
* Remove the commit
* Keep it like that
* Convert that in scala (in this case this is time consuming, and should be done in a separate ticket IMO).

Please me what you prefer.